app/controllers/teams_controller.rb

```diff
@@ -106,4 +101,23 @@ def set_display_roles
       @display_roles = ['student']
       @display_roles.map!(&:pluralize)
     end
+
+    def show_teams_by(params)
```

Comment:
👍 sure, the eager loading should be independent of whether there's a `:sort` param or not - the `last-activity` timestamp is still displayed. Don't see the reason why it should be different @carpodaster.

Why we need to eager load all activities to then in the end just pluck a single timestamp per team is a different thing though 😉